subcommands/devices/show.go

```diff
@@ -49,6 +49,9 @@ func doShow(cmd *cobra.Command, args []string) {
 	fmt.Printf("Up to date:\t%v\n", device.UpToDate)
 	fmt.Printf("Target:\t\t%s / sha256(%s)\n", device.TargetName, device.OstreeHash)
 	fmt.Printf("Ostree Hash:\t%s\n", device.OstreeHash)
+	if len(device.LmpVer) > 0 {
+		fmt.Printf("LmP Version:\t%s\n", device.LmpVer)
```

Comment:
I am not sure about it, at least in scope of this PR/effort.

I think it might be useful to filter out (and/or sort out) devices by a specific lmp version.

Let's roll out this change at first, get some feedback, and then decide on the lmp version application to the devices list command.